src/breadcrumb/__tests__/breadcrumb.spec.ts

```diff
@@ -39,10 +49,46 @@ describe('Breadcrumb', () => {
   });
 
   it('should be hidden with `autoHideContainer`', () => {
+    const render = createRenderer({
+      defaultState,
+      template: '<ais-breadcrumb></ais-breadcrumb>',
+      TestedWidget: NgAisBreadcrumb,
+    });
     const fixture = render({ items: [] });
     fixture.componentInstance.testedWidget.autoHideContainer = true;
     fixture.detectChanges();
 
     expect(fixture).toMatchSnapshot();
   });
+
+  it('should create widget with connectBreadcrumb and pass instance options', () => {
+    const createWidget = jest.spyOn(NgAisBreadcrumb.prototype, 'createWidget');
+
+    const attributes = ['attr.lvl1', 'attr.lvl2'];
+    const transformItems = jest.fn(x => x);
+
+    const render = createRenderer({
+      defaultState,
+      template: `
+        <ais-breadcrumb 
+          [attributes]="attributes"
+          rootPath="Audio" 
+          separator=" / " 
+          [transformItems]="transformItems">
+        </ais-breadcrumb>`,
+      TestedWidget: NgAisBreadcrumb,
+      methods: { transformItems, attributes },
```

Comment:
From what I've understood this parameter is used to expose values to the template correct? If I'm correct we should rename it to something more explicit. It tooks me some time to understand the purpose of the option.

Reply:
it's to add things to the prototype of the created class, and also used / similar in the stories. 

something like `additionalProperties` could _maybe_ work

Reply:
yes, I'll wait for all the PRs using it to be merged and bulk rename it.
It was originally copied from storybook helper and intended to pass methods.

Reply:
opened an issue for it https://github.com/algolia/angular-instantsearch/issues/580